test/e2e/service_test.go

```diff
@@ -0,0 +1,72 @@
+// Copyright 2021 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package e2e
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	corev1 "k8s.io/api/core/v1"
+)
+
+// TestClusterIPHostAccess tests traffic from host to Cluster IP Service.
+func TestClusterIPHostAccess(t *testing.T) {
+	skipIfProviderIs(t, "kind", "Skipping Kind provider for now.")
```

Comment:
Updated. I have created 2 `TestClusterIPHostAccess*` functions for kind and others. Thanks to your comment below, I decided to curl from hostnetwork pod for kind. For other providers, since ssh-config change is already needed for windows nodes to run command on host, then linux nodes do the same.